extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -80,28 +80,19 @@ public boolean test(DotName dotName) {
 
     // Types ignored for reflection used by the RESTEasy and SmallRye REST client extensions.
     private static final Set<DotName> TYPES_IGNORED_FOR_REFLECTION = new HashSet<>(Arrays.asList(
-            // Jackson
-            DotName.createSimple("com.fasterxml.jackson.databind.JsonNode"),
+    // Consider adding packages below instead if it makes more sense
+    ));
 
+    private static final String[] PACKAGES_IGNORED_FOR_REFLECTION = {
+            // JSON-P
+            "javax.json.",
+            // Jackson
+            "com.fasterxml.jackson.databind.",
             // JAX-RS
-            DotName.createSimple("javax.ws.rs.core.Response"),
-            DotName.createSimple("javax.ws.rs.core.Response.StatusType"),
-            DotName.createSimple("javax.ws.rs.container.AsyncResponse"),
```

Comment:
This one is missing, it's on the `javax.ws.rs.container` package, not on `javax.ws.rs.core`

Reply:
Good catch, fixed!